docs/contributing-to-airbyte/README.md

```diff
@@ -4,149 +4,35 @@ description: 'We love contributions to Airbyte, big or small.'
 
 # Contributing to Airbyte
 
-Thank you for your interest in contributing! We love community contributions. Contribution guidelines are listed below. If you're unsure about how to start contributing or have any questions even after reading them, feel free to ask us on [Slack](https://slack.airbyte.io) in the \#help-contributions.
+Thank you for your interest in contributing! We love community contributions. Contribution guidelines are listed below. If you're unsure about how to start contributing or have any questions even after reading them, feel free to ask us on [Slack](https://slack.airbyte.io) in the `#contributions` channel.
 
 However, for those who want a bit more guidance on the best way to contribute to Airbyte, read on. This document will cover what we're looking for. By addressing the points below, the chances that we can quickly merge or address your contributions will increase.
 
-## Code of conduct
+We appreciate first time contributors and we are happy to assist you in getting started. In case of questions, just reach out to us via [email](mailto:hey@airbyte.io) or [Slack](https://slack.airbyte.io)! Here is a list of easy [good first issues](https://github.com/airbytehq/airbyte/labels/good%20first%20issue) to do if you're hunger to start contributing.
+## Code of Conduct
 
-Please follow our [Code of conduct](code-of-conduct.md) in the context of any contributions made to Airbyte.
+Please follow our [Code of Conduct](code-of-conduct.md) in the context of any contributions made to Airbyte.
```

Comment:
Before getting started, please review Airbyte's Code of Conduct.  Everyone interacting in Slack, codebases,  mailing lists, events, or other Airbyte activities is expected to follow [Code of Conduct](code-of-conduct.md).